schema/data/portal/form_result.sql

```diff
@@ -23,5 +23,6 @@ setval('ggircs_portal.form_result_id_seq', (select max(id) from ggircs_portal.fo
 where setval = 0;
 
 select ggircs_portal.create_application_mutation_chain(2);
+update ggircs_portal.application set reporting_year = 2019 where id = 2;
```

Comment:
Add TODO: create_application_mutation_chain should set the reporting_year to the current reporting year